kolibri/plugins/user/assets/src/views/sign-in-page/index.vue

```diff
@@ -201,9 +212,27 @@
       versionMsg() {
         return this.$tr('poweredBy', { version: __version });
       },
+      hasServerError() {
+        return !!(this.passwordMissing || this.invalidCredentials);
```

Comment:
Cast it to a Boolean instead.

Reply:
but https://user-images.githubusercontent.com/25045166/35070604-a1aad520-fb92-11e7-8423-feeeacfdfd25.png

Reply:
I agree though